services/help-service.ts

```diff
@@ -137,7 +171,22 @@ export class HelpService implements IHelpService {
 	private tryOpeningSelectedPage(htmlPage: string): boolean {
 		const fileList = this.$fs.enumerateFilesInDirectorySync(this.pathToHtmlPages);
 		this.$logger.trace("File list: " + fileList);
-		const pageToOpen = _.find(fileList, file => path.basename(file) === htmlPage);
+		let pageToOpen = _.find(fileList, file => path.basename(file) === htmlPage);
+		if (!pageToOpen) {
+			const installedExtensionsData = this.$extensibilityService.getInstalledExtensionsData();
+
+			for (const extensionData of installedExtensionsData) {
```

Comment:
You can extract [this](https://github.com/telerik/mobile-cli-lib/blob/9412c49b11f6a69b58b6970a9428e916c6bda736/services/help-service.ts#L178-L188) as a separate function and reuse it in `tryOpeningSelectedPage` and in `readMdFileForCommand`